lib/grammers-client/src/lib.rs

```diff
@@ -46,3 +46,9 @@ pub(crate) mod utils;
 
 pub use client::{Client, Config, InitParams, SignInError};
 pub use types::{button, reply_markup, ChatMap, InputMessage, Update};
+
+pub use grammers_crypto;
```

Comment:
I'm not sure we should re-export this. When would a user need it when using the client?

Reply:
I reexported everything, just in case. Personally I only need `session`, `tl-types` and `mtsender`. I'm happy to remove this.